SConstruct

```diff
@@ -381,7 +381,8 @@ if env["prereqs"]:
 	CheckAsio(conf) & \
 	conf.CheckBoost("thread") & \
         conf.CheckBoost("locale") & \
-        conf.CheckBoost("filesystem") \
+        conf.CheckBoost("filesystem") & \
+        conf.CheckBoost("graph") \
```

Comment:
Should be part of `have_client_prereqs` below.

Reply:
Done! Thank you!